locales/en/app.ftl

```diff
@@ -853,6 +853,7 @@ ad-unit-6-before-you-complete = Before you complete that next signup, use an ema
 ## The following messages are brands and should be kept entirely in English
 
 -brand-firefox = Firefox
+-brand-monitor = Monitor
```

Comment:
Not sure if one is better than the other, or if there are potential issues w/ having the same value in multiple aliases, but 🤷 

https://github.com/mozilla/blurts-server/blob/e793dd93f63327c8d8026d899679b226c98a405d/locales/en/app.ftl#L8

Reply:
That term definition in the section that should be removed (v1) :-(